README.rst

```diff
@@ -6,19 +6,20 @@ audformat
 
 Specification and reference implementation of **audformat**.
 
-audformat is intended to store media data,
+audformat stores media data,
 such as audio or video,
 together with corresponding annotations
 in a pre-defined way.
-An audformat database is a folder that contains
-media files together with a header YAML file
-and one or several files storing the annotations.
+This makes it easy to combine or replace databases
+in machine learning projects.
 
-An audformat database is represented as a
-``audformat.Database`` object and can be loaded with
-``audformat.Database.load()``
-or written to disk with
-``audformat.Database.save()``.
+An audformat database is a folder
+that contains media files
+together with a header YAML file
+and one or several files storing the annotations.
+The database is represented as an ``audformat.Database`` object
```

Comment:
```suggestion
The database is represented as an :class:`audformat.Database` object
```

Reply:
I didn't used `:class:` here as it will not be rendered correctly in the README presented at https://github.com/audeering/audformat but only in the rendered docuemntation.

For the same reason I stopped using it in the CHANGELOG for projects on Github. Because otherwise it would also look weird at https://github.com/audeering/audformat/releases and https://pypi.org/project/audformat/

Reply:
Oh right.